plugins/lsp/language-servers/rust-analyzer.nix

```
@@ -30,6 +30,16 @@ in
 
     # TODO: make nullable
     rustcPackage = mkPackageOption pkgs "rustc" { };
+
+    installRustfmt = mkOption {
+      type = with types; nullOr bool;
+      default = null;
+      example = true;
+      description = "Whether to install `rustfmt`.";
+    };
+
+    # TODO: make nullable
+    rustfmtPackage = mkPackageOption pkgs "rustfmt" { };
```

Any reason this shouldn't be nullable yet?


```suggestion
    rustfmtPackage = mkPackageOption pkgs "rustfmt" {
      nullable = true;
    };
```

---

Ah, I guess this is because we have the `install` option too? So the package option being nullable is kinda redundant...

At least recently, we typically skip the install-toggle-options and instead just make the installable thing nullable. Either null by default with a useful example, or set to something by default but documented as nullable.

In this case I don't have a strong preference, since it matches the style of this module's other options.